app/src/androidTest/java/org/mozilla/focus/helpers/TestHelper.kt

```diff
@@ -176,66 +174,12 @@ object TestHelper {
     }
 
     fun verifyDownloadedFileOnStorage(fileName: String) {
-        val context = getInstrumentation().targetContext
-        val resolver = context.contentResolver
-        val fileUri = queryDownloadMediaStore(fileName)
-
-        val fileExists: Boolean = fileUri?.let {
-            resolver.openInputStream(fileUri).use {
-                it != null
-            }
-        } ?: false
-
-        assertTrue(fileExists)
-    }
-
-    /**
-     * Check the "Downloads" public directory for [fileName] and returns an URI to it.
-     * May be `null` if there is no file with that name in "Downloads".
-     */
-    @TargetApi(Build.VERSION_CODES.Q)
-    private fun queryDownloadMediaStore(fileName: String): Uri? {
-        val context = getInstrumentation().targetContext
-        val resolver = context.contentResolver
-        val queryProjection = arrayOf(MediaStore.Downloads._ID)
-        val querySelection = "${MediaStore.Downloads.DISPLAY_NAME} = ?"
-        val querySelectionArgs = arrayOf(fileName)
-
-        val queryBundle = Bundle().apply {
-            putString(ContentResolver.QUERY_ARG_SQL_SELECTION, querySelection)
-            putStringArray(ContentResolver.QUERY_ARG_SQL_SELECTION_ARGS, querySelectionArgs)
-        }
-
-        // Query if we have a pending download with the same name. This can happen
-        // if a download was interrupted, failed or cancelled before the file was
-        // written to disk. Our logic above will have generated a unique file name
-        // based on existing files on the device, but we might already have a row
-        // for the download in the content resolver.
-        val collection = MediaStore.Downloads.getContentUri(MediaStore.VOLUME_EXTERNAL_PRIMARY)
-        val queryCollection =
-            if (SDK_INT >= Build.VERSION_CODES.R) {
-                queryBundle.putInt(MediaStore.QUERY_ARG_MATCH_PENDING, MediaStore.MATCH_INCLUDE)
-                collection
-            } else {
-                @Suppress("DEPRECATION")
-                setIncludePending(collection)
-            }
-
-        var downloadUri: Uri? = null
-        resolver.query(
-            queryCollection,
-            queryProjection,
-            queryBundle,
-            null,
-        )?.use {
-            if (it.count > 0) {
-                val idColumnIndex = it.getColumnIndex(MediaStore.Downloads._ID)
-                it.moveToFirst()
-                downloadUri = ContentUris.withAppendedId(collection, it.getLong(idColumnIndex))
-            }
-        }
-
-        return downloadUri
+        val storageManager: StorageManager? =
+            getInstrumentation().targetContext.getSystemService() as StorageManager?
+        val storageVolumes = storageManager!!.storageVolumes
+        val storageVolume: StorageVolume = storageVolumes.get(0)
```

Comment:
nit: An indexing operator might be more idiomatic.

```suggestion
        val storageVolume: StorageVolume = storageVolumes[0]
```